### DISCUSSION TITLE
Il sito opendatasicilia è ora attivo sul nuovo server

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/opendatasicilia.it#44] [Annunci]
Buonasera a tutte e tutti,
il sito opendatasicilia è ora attivo sul nuovo server.

Le novità attuali da sottolineare sono:

- una version di wordpress aggiornata;
- il certificato che abilita l'HTTPS;
- c'è anche un grafica nuova, ma è soltanto temporanea.

Non è ancora il nuovo sito in termini complessivi, ma lo è in termini di motore/back-end. 
Sul front-end i lavori sono in corso.

L'URL è sempre https://opendatasicilia.it/

Se vedete ancora la vecchia grafica, vuol dire che ancora c'è una cache di DNS (tra un po' si svouterà).

Saluti

**Comment by dennisangemi** [comment]
Grazie mille @aborruso . Chi aveva modificato il file di host deve rimuovere le modifiche?

**Reply by aborruso** [reply]
> Grazie mille @aborruso . Chi aveva modificato il file di host deve rimuovere le modifiche?

yes

**Reply by gabacode** [reply]
@aborruso mi sa che c'e' un link "vecchio" che non permette di fare il build: https://i2.wp.com/opendatasicilia.it/wp-content/uploads/2015/02/2015-02-22_13h23_57.jpg?resize=1%2C1&#038;ssl=1 - si dovrebbe vedere nel db se ci sono altre istanze di "i2.wp.com"

**Reply by aborruso** [reply]
@gabacode se c'è un URL sbagliato, non viene compilato il "tuo" lato?

Vado a vedere di che si tratta in ogni caso

**Reply by aborruso** [reply]
> @aborruso mi sa che c'e' un link "vecchio" che non permette di fare il build: https://i2.wp.com/opendatasicilia.it/wp-content/uploads/2015/02/2015-02-22_13h23_57.jpg?resize=1%2C1&#038;ssl=1 - si dovrebbe vedere nel db se ci sono altre istanze di "i2.wp.com"

Ho cercato nei post content e non c'è. Per stasera mi devo fermare.
Ma ci torno

**Reply by aborruso** [reply]
@gabacode ho esportato tutto il db, in formato SQL e non ho traccia di  `https://i2.wp.com`

E non mi stranizza. Se vedi bene non è URL vecchio, ma un URL di qualcosa di wordpress.com, qualcosa che fa caching delle immagini probabilmente. Ma ripeto, nel db non ne trovo traccia.
Tu dove ne hai traccia?

**Reply by gabacode** [reply]
> se c'è un URL sbagliato, non viene compilato il "tuo" lato?

Volevo provare ad ovviare con try and catch ma non funziona e leggo che non è best practice, devo ancora capire bene da dove viene l'errore e lavorerò ad una fix

> @gabacode ho esportato tutto il db, in formato SQL e non ho traccia di `https://i2.wp.com`
> 
> E non mi stranizza. Se vedi bene non è URL vecchio, ma un URL di qualcosa di wordpress.com, qualcosa che fa caching delle immagini probabilmente. Ma ripeto, nel db non ne trovo traccia. Tu dove ne hai traccia?

Effettivamente il file incriminato dovrebbe essere in [questo post](https://opendatasicilia.it/wp-admin/post.php?post=556&action=edit
), ma vedo che ha già l'url nuovo. Per qualche motivo gatsby mi fetcha i link vecchi, probabilmente qualche cache da qualche parte. Riprovo domani,
Grazie!

**Reply by gabacode** [reply]
@aborruso dopo qualche ora la situazione si e' sistemata! Ho fatto un build su [gh-pages](https://github.com/opendatasicilia/opendatasicilia.it/tree/gh-pages)
